edtf/parser/tests.py

```diff
@@ -111,29 +111,35 @@
     ('2010-24', ('2010-12-01', '2010-12-31')),
 
     # ******************************* LEVEL 2 *********************************
-    # Partial Uncertain/Approximate
+    # Qualification
+    # Group qualification: a qualification character to the immediate right of a component applies
+    # to that component as well as to all components to the left.
+    # year, month, and day are uncertain and approximate
+    ('2004-06-11%', ('2004-06-11', '2004-06-09', '2004-06-13')),
```

Comment:
Shouldn't this one be a broader range, given the year and month are also uncertain and approximate?

Reply:
You can say `2002%-06-11` instead if you want the narrow range.  So I think we should fuzz all parts to the left of the % symbol, and we'd get the broader "2002-04-09" - "2006-08-13" range.